test/cppsuite/src/main/test.cpp

```diff
@@ -26,19 +26,10 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
-/* Required to build using older versions of g++. */
```

Comment:
As discussed let's confirm that we have docs specifying a min g++ version. If we don't (or we support versions that need this include) we should leave it alone.

Reply:
Our docs say GCC 8.5+ on [this page](https://source.wiredtiger.com/develop/build-posix.html)

Reply:
I verified I could compile with gcc-7

Reply:
Sounds good to me. We formally support back to 8.5 and unofficially back to 7. Anything older that breaks with this change is well beyond our support window